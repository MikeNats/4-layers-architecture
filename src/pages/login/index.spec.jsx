import React from 'react'
import LogInFormPage from '.'
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

jest.mock('../../services/transactions/actions');

Enzyme.configure({ adapter: new Adapter() })

describe("LogInFormPage", () => {
  describe('component', () => {
    let component
    it('renders as expected', () => {

      component = shallow(<LogInFormPage /> );
      expect(component).toMatchSnapshot();
   });
   it('should set to props', () => {	
		  component = shallow(<LogInFormPage />).find('main');

     expect(component.props()).toHaveProperty('className');
     expect(component.prop("className")).toEqual('base-layout horiz-vertic-align-layout');
    });
  });
});
 
