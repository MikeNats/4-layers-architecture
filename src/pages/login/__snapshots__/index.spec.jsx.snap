// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogInFormPage component renders as expected 1`] = `ShallowWrapper {}`;
